.env

```diff
@@ -2,15 +2,7 @@ FLASK_APP=run.py
 SQLALCHEMY_DATABASE_URI=postgresql://user_name:change_password@resources-psql:5432/resources-psql
```

Comment:
Can you have the application build SQLALCHEMY_DATABASE_URI from the provided variables?